### DISCUSSION TITLE
Is there a way to create a simple weekly / monthly dashboard for diaper and feed count?

### DISCUSSION THREAD

**Discussion by kmanan** [babybuddy/babybuddy#614] [Support]
I have BB as an addon on Home Assistant. I am currently tracking extremely basic count of diapers changed in a day and feeds in a day (no type / quantity / source of feed etc.) I would like to create a simple dashboard that answers:
1. Bar chart showing how many diapers have been changed in the week as a per day count (days on the x axis and count on the Y axis)
2. Same as above for feeds

I am not a db developer and have very limited knowledge of BB and HA. Was wondering I could follow some guidance and find my way around. Any help would be appreciated. Thanks

**Comment by cdubz** [comment]
The dashboard is mostly quicker “at a glance” data. It sounds you are looking for the reports. Click the child’s name and then the little graph icon to get a list of all reports. Sounds like you’re looking for Diaper Change Types and Feeding Amounts.

The reports are a little too hidden in the app and they are missed in the user guide documentation.

**Reply by kmanan** [reply]
> The dashboard is mostly quicker “at a glance” data. It sounds you are looking for the reports. Click the child’s name and then the little graph icon to get a list of all reports. Sounds like you’re looking for Diaper Change Types and Feeding Amounts.
> 
> The reports are a little too hidden in the app and they are missed in the user guide documentation.

Yes, I think something like a report. I saw the Reports section and it wasn't getting me what I wanted, at least for feeding. For diapers, Diaper Change type report is close to what I am looking for feeding too.
